webpack.config.js

```diff
@@ -26,8 +26,6 @@ const WC_ADMIN_CONFIG = require( path.join( __dirname, 'config', WC_ADMIN_PHASE
 const externals = {
 	'@wordpress/api-fetch': { this: [ 'wp', 'apiFetch' ] },
 	'@wordpress/blocks': { this: [ 'wp', 'blocks' ] },
-	'@wordpress/components': { this: [ 'wp', 'components' ] },
```

Comment:
I'm not sure if removing these has other implications, but this prevented new components from working due to missing functions.

Reply:
I'm not sure about this either. Things seem to be working well for me though with this change.